pennylane/templates/broadcast.py

```diff
@@ -576,18 +558,19 @@ def circuit(pars):
 
     #########
 
-    # define wire sequence for patterns
+    # define wire sequences for patterns
     wire_sequence = {
-        "single": wires,
-        "double": [[wires[i], wires[i + 1]] for i in range(0, len(wires) - 1, 2)],
-        "double_odd": [[wires[i], wires[i + 1]] for i in range(1, len(wires) - 1, 2)],
-        "chain": [[wires[i], wires[i + 1]] for i in range(len(wires) - 1)],
+        "single": [wires[i] for i in range(len(wires))],
```

Comment:
How come the list comprehension was needed here? To have a list of `Wires` objects each containing exactly one element?

Reply:
Yes exactly!